setup/09-tradeoffs/movr/pricing/data/pricing_database.sql

```diff
@@ -0,0 +1,26 @@
+CREATE DATABASE movr_pricing;
+
+ALTER DATABASE movr_pricing SET PRIMARY REGION "us-east";
+ALTER DATABASE movr_pricing ADD REGION "us-central";
+ALTER DATABASE movr_pricing ADD REGION "us-west";
+ALTER DATABASE movr_pricing ADD REGION "eu-west";
+
+CREATE TABLE movr_pricing.promo_codes (
+    code STRING PRIMARY KEY,
+    description STRING NOT NULL, 
+    creation_time TIMESTAMP NOT NULL, 
+    expiration_time TIMESTAMP NOT NULL, 
+    rules JSONB NOT NULL
+);
+
+CREATE TABLE movr_pricing.vip_rates (
+    rate_code STRING PRIMARY KEY,
+    market STRING NOT NULL,
+    description STRING NOT NULL, 
+    creation_time TIMESTAMP NOT NULL 
+);
+
+ALTER TABLE movr_pricing.promo_codes SET locality GLOBAL;
+ALTER TABLE movr_pricing.vip_rates SET LOCALITY REGIONAL BY TABLE IN "us-west";
+
+ALTER DATABASE movr_pricing PLACEMENT RESTRICTED;
```

Comment:
FYI currently this is part of the exercise instructions (as opposed to part of the setup)

Reply:
The exercise for this one should ask students to change this from placement `restricted` to `default`
I'll check on that!